src/screencast/screencast_common.c

```diff
@@ -53,25 +54,75 @@ struct wl_buffer *import_wl_shm_buffer(struct xdpw_screencast_instance *cast, in
 	return buffer;
 }
 
-enum spa_video_format xdpw_format_pw_from_wl_shm(enum wl_shm_format format) {
+enum wl_shm_format xdpw_format_wl_shm_from_drm_fourcc(uint32_t format) {
+	switch (format) {
```

Comment:
This can be simplified: https://gitlab.freedesktop.org/wlroots/wlroots/-/blob/bcefb71cf61b4d12465c8ba41c79ec6b651e045e/render/pixel_format.c#L146

Reply:
Nice!